docs/cep/index.md

```diff
@@ -28,6 +28,90 @@ You can process streams to perform the following actions with your data:
 - Extract, transform, and load (ETL) collections, tailing files, and scraping HTTP endpoints.
 - Integrating stream data and trigger actions based on the data. This can be a single service request or a complex enterprise integration flow.
 
+## Stream Worker Explanation
+
+This section explains the parts of this stream worker and what they are doing.
+
+### Metadata
+
+This information defines basic information about the stream worker. Every stream worker must have at least a name and query language version in order to be valid.
+
+- **Name** - `@App:name("ExampleApp")`
+- **Query language version (optional)** - @App:qlVersion("2")
+- **Description (optional)** - @App:description('An application for enriching transactions.')
+- **Other information (optional)** - By convention, you can enter a comment with testing information, update logs, or other useful information at the beginning of the stream worker definition between `/**` and `**/`. This is similar to a docstring in functions.
+
+### Input and Output
```

Comment:
Let's narrow this a bit for now, just link to the Sink and Source sections, because they are the only two that are actually required. Table won't always be present, as I understand it. (ARGH, we need to get with @StoyanTc on this.)